war/src/test/java/io/lumeer/engine/controller/CollectionMetadataFacadeIntegrationTest.java

```diff
@@ -231,39 +209,21 @@ public void testGetInternalCollectionName() throws Exception {
    }
 
    @Test
-   public void testCollectionMetadataCollectionName() {
-      assertThat(collectionMetadataFacade.collectionMetadataCollectionName("collection")).isEqualTo("meta.collection");
-   }
-
-   @Test
    public void testCreateInitialMetadata() throws Exception {
       String collection = internalName(COLLECTION_CREATE_INITIAL_METADATA);
-      String metaCollection = collectionMetadataFacade.collectionMetadataCollectionName(collection);
-
-      // set up collection (delete one created during previous test)
-      dataStorage.dropCollection(metaCollection);
-
-      dataStorage.createCollection(metaCollection);
       collectionMetadataFacade.createInitialMetadata(collection, COLLECTION_CREATE_INITIAL_METADATA);
 
-      String name = collectionMetadataFacade.getOriginalCollectionName(collection);
-      String lock = collectionMetadataFacade.getCollectionLockTime(collection);
-
-      assertThat(name).isEqualTo(COLLECTION_CREATE_INITIAL_METADATA);
-      assertThat(lock).isEmpty();
-   }
-
-   @Test
-   public void testSetGetCollectionLockTime() throws Exception {
-      setUpCollection(COLLECTION_SET_LOCK_TIME);
-
-      collectionFacade.createCollection(COLLECTION_SET_LOCK_TIME);
-      String collection = internalName(COLLECTION_SET_LOCK_TIME);
-      String time = Utils.getCurrentTimeString();
-      collectionMetadataFacade.setCollectionLockTime(collection, time);
-      String timeTest = collectionMetadataFacade.getCollectionLockTime(collection);
-
-      assertThat(time).isEqualTo(timeTest);
+      CollectionMetadata metadata = collectionMetadataFacade.getCollectionMetadata(collection);
+
+      assertThat(metadata.getName()).isEqualTo(COLLECTION_CREATE_INITIAL_METADATA);
+      assertThat(metadata.getInternalName()).isEqualTo(collection);
+      assertThat(metadata.getProjectId()).isEqualTo(projectFacade.getCurrentProjectId());
+      assertThat(metadata.getAttributes()).isEmpty();
+      assertThat(metadata.getLastTimeUsed()).isNotEmpty();
+      assertThat(metadata.getRecentlyUsedDocumentIds()).isEmpty();
+      assertThat(metadata.getCreateDate()).isNotEmpty();
+      assertThat(metadata.getCreator()).isEqualTo(userFacade.getUserEmail());
+      assertThat(metadata.getCustomMetadata()).isEmpty();
```

Comment:
[Soft assertions](http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#soft-assertions) would be better here.